dnsaas/admin.py

```diff
@@ -0,0 +1,427 @@
+from django_extensions.admin import ForeignKeyAutocompleteAdmin
+from django.contrib import admin
+from django.contrib.admin.sites import AdminSite
+from django.forms import (
+    HiddenInput,
+    NullBooleanSelect,
+    ModelForm,
+    ValidationError,
+    ModelChoiceField,
+)
+from powerdns.models.templates import (
+    DomainTemplate,
+    RecordTemplate,
+)
+
+from powerdns.models.powerdns import (
+    CryptoKey,
+    Domain,
+    DomainMetadata,
+    Record,
+    SuperMaster,
+)
+from powerdns.models.tsigkeys import TsigKey
+from powerdns.utils import Owned, DomainForRecordValidator, is_owner
+
+
+admin_site2 = AdminSite('admin2')
+
+
+#TODO: is this really needed?
+class CopyingAdmin(admin.ModelAdmin):
```

Comment:
try to keep this admin as simple as possible - i'd not use such classes (copying + owned)